code/datums/mutations.dm

```diff
@@ -236,8 +236,8 @@
 		owner.visible_message("<span class='danger'>[owner] starts having a seizure!</span>", "<span class='userdanger'>You have a seizure!</span>")
 		owner.Paralyse(10)
 		owner.Jitter(1000)
-		spawn(90)
-			owner.jitteriness = 10
+		sleep(90)
+		owner.jitteriness = 10
```

Comment:
unless somebody made on_life a `set waitfor = 0` proc, this would block all life() processing on this mob (it wouldn't block the mob controller, but it would block the rest of the life processing for that indivisual mob)